apscheduler/util.py

```diff
@@ -4,7 +4,7 @@
 from calendar import timegm
 from inspect import signature
 from functools import partial
-from inspect import isclass, ismethod
+from inspect import isclass, ismethod, iscoroutinefunction
```

Comment:
You need to use `asyncio.iscoroutinefunction()` here so it catches coroutines created in asyncio debug mode.

Reply:
I've explicitly made it say asyncio.iscoroutinefunction in situ so readers know precisely which version is being called. Not clear this is about debug mode so much as decorator-based async functions ala @coroutine.